dbmodels/link.go

```diff
@@ -36,9 +36,9 @@ type CLAOfLink struct {
 }
 
 type OrgRepo struct {
-	Platform string `json:"platform"`
-	OrgID    string `json:"org_id"`
-	RepoID   string `json:"repo_id"`
+	Platform string `json:"platform" required:"true"`
+	OrgID    string `json:"org_id" required:"true"`
+	RepoID   string `json:"repo_id" required:"true"`
```

Comment:
repo is not required. Besides, you should think about the other use cases of OrgRepo. It is not good to chage it directly